packages/ui-components/src/components/Dialog.js

```diff
@@ -72,7 +79,7 @@ export const DialogHeader = ({ title, onClose, color, children }) => (
 );
 
 DialogHeader.propTypes = {
-  title: PropTypes.string.isRequired,
+  title: PropTypes.string,
```

Comment:
It's minor and we don't have consistency yet but we should eventually have prop types everywhere. Could you add the title default below.